crates/nu-engine/src/glob_from.rs

```diff
@@ -6,7 +6,7 @@ use std::{
     path::{Component, Path, PathBuf},
 };
 
-const GLOB_CHARS: &[char] = &['*', '?', '['];
+pub const GLOB_CHARS: &[char] = &['*', '?', '['];
```

Comment:
I think `GLOB_CHARS` is an implementation detail. It's probably better to duplicate this in your code than making it public.

Alternatively, maybe `s == Pattern::escape(s)` is a better way to check if `s` looks like a glob pattern?